File: app/src/main/java/vn/loitp/app/activity/customviews/wwlvideo/FragmentHost.java
package vn.loitp.app.activity.customviews.wwlvideo;

import vn.loitp.app.activity.customviews.wwlvideo.utils.Dataset;

/**
 * Created by thangn on 2/26/17.
 */

public interface FragmentHost {
    void goToDetail(Dataset.DatasetItem item);

    void onVideoCollapse();

    void onVideoFullscreen(boolean selected);
}

File: app/src/main/java/vn/loitp/app/a/customviews/wwlVideo/interfaces/FragmentHost.kt
package vn.loitp.app.a.customviews.wwlVideo.interfaces

import vn.loitp.app.a.customviews.wwlVideo.utils.WWLVideoDataset

interface FragmentHost {
    fun goToDetail(item: WWLVideoDataset.DatasetItem)

    fun onVideoCollapse()

    fun onVideoFullscreen(selected: Boolean)
}
